crates/matrix-sdk/README.md

```diff
@@ -59,18 +59,20 @@ More examples can be found in the [examples] directory.
 
 The following crate feature flags are available:
 
-| Feature            | Default | Description                                                    |
-| ------------------ | :-----: | -------------------------------------------------------------- |
-| `anyhow`           |   No    | Better logging for event handlers that return `anyhow::Result` |
-| `encryption`       |   Yes   | End-to-end encryption support                                  |
-| `eyre`             |   No    | Better logging for event handlers that return `eyre::Result`   |
-| `image_proc`       |   No    | Enables image processing to generate thumbnails                |
-| `image_rayon`      |   No    | Enables faster image processing                                |
-| `markdown`         |   No    | Support to send Markdown-formatted messages                    |
-| `qrcode`           |   Yes   | QR code verification support                                   |
-| `sled_cryptostore` |   Yes   | Persistent storage for E2EE related data                       |
-| `socks`            |   No    | Enables SOCKS support in the default HTTP client, [`reqwest`]  |
-| `sso_login`        |   No    | Enables SSO login with a local HTTP server                     |
+| Feature            | Default | Description                                                           |
+| ------------------ | :-----: | --------------------------------------------------------------------- |
+| `anyhow`           |   No    | Better logging for event handlers that return `anyhow::Result`        |
+| `encryption`       |   Yes   | End-to-end encryption support                                         |
+| `eyre`             |   No    | Better logging for event handlers that return `eyre::Result`          |
+| `image_proc`       |   No    | Enables image processing to generate thumbnails                       |
+| `image_rayon`      |   No    | Enables faster image processing                                       |
+| `markdown`         |   No    | Support to send Markdown-formatted messages                           |
+| `qrcode`           |   Yes   | QR code verification support                                          |
+| `sled_cryptostore` |   Yes   | Persistent storage for E2EE related data                              |
```

Comment:
I think we should just merge those sled features.

Reply:
> didn't find a way to only enable a crate's feature if the crate is already enabled

This will become possible in the next Rust release: https://doc.rust-lang.org/cargo/reference/unstable.html#weak-dependency-features

Reply:
Alright, let's leave it as is for now, and consider bumping MSRV to get this in the future.